MinecraftClient/McClient.cs

```diff
@@ -292,8 +290,9 @@ private void CommandPrompt()
                         switch (command[0].ToLower())
                         {
                             case "autocomplete":
-                                if (command.Length > 1) { ConsoleIO.WriteLine((char)0x00 + "autocomplete" + (char)0x00 + handler.AutoComplete(command[1])); }

```

Comment:
`autocomplete` is a command to interact with the GUI so it should not be translated.
The `\0command\0args` are used for passing messages between GUI and MCC.